db/seeds.rb

```diff
@@ -5,3 +5,19 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+User.create!(email: 'email@example.com', password: 'password', password_confirmation: 'password')
+
+Development.create!(
+  [
+    { application_number: '193665776',
```

Comment:
Just to help make seed data look a tiny bit more real, Southwark's application numbers look like `AP/19/1234'. It's always `AP`, then 2 digit year of application, then 4 numbers.